CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
```

Comment:
This file already exists at /.github/CODEOWNERS. See #12

Reply:
I overlooked that one! Only saw it in #1. Di not notice that it's in a different location. The existing one is better then this. I'll adjust the PR.